applications/ConvectionDiffusionApplication/custom_elements/symbolic_qs_convection_diffusion_explicit.cpp

```diff
@@ -0,0 +1,926 @@
+            rhs[1]=-DN_DX_1_0*crhs35 - DN_DX_1_1*crhs36 - DN_DX_1_2*crhs37 + crhs101*crhs126 - crhs108*crhs123 - crhs110*crhs124 - crhs112*crhs125 - crhs113*crhs126 - crhs114*crhs123 - crhs115*crhs124 - crhs116*crhs125 - crhs117*crhs126 + crhs118 + crhs119 + crhs120 + crhs121 + crhs122 - crhs123*crhs77 + crhs123*crhs82 + crhs123*crhs88 - crhs124*crhs31 + crhs124*crhs90 + crhs124*crhs93 - crhs125*crhs74 + crhs125*crhs95 + crhs125*crhs98 - crhs126*crhs71 + crhs126*crhs99 - 0.5854102*crhs15 - 0.5854102*crhs23 - 0.5854102*crhs31 + crhs33 + crhs34 + crhs42 + crhs45 + crhs52 + crhs55 + crhs62 + crhs65 - 0.5854102*crhs7 + crhs72 + crhs75 + 0.40000000301872*f[1];
+            rhs[2]=-DN_DX_2_0*crhs35 - DN_DX_2_1*crhs36 - DN_DX_2_2*crhs37 + crhs101*crhs131 - crhs108*crhs128 - crhs110*crhs129 - crhs112*crhs130 - crhs113*crhs131 - crhs114*crhs128 - crhs115*crhs129 - crhs116*crhs130 - crhs117*crhs131 + crhs127 - crhs128*crhs77 + crhs128*crhs82 + crhs128*crhs88 - crhs129*crhs31 + crhs129*crhs90 + crhs129*crhs93 - crhs130*crhs74 + crhs130*crhs95 + crhs130*crhs98 - crhs131*crhs71 + crhs131*crhs99 + crhs42 - 0.5854102*crhs44 + crhs52 - 0.5854102*crhs54 + crhs62 - 0.5854102*crhs64 + crhs72 - 0.5854102*crhs74 + 0.40000000301872*f[2] + 0.19999999899376*f[3];
+            rhs[3]=-DN_DX_3_0*crhs35 - DN_DX_3_1*crhs36 - DN_DX_3_2*crhs37 + crhs101*crhs135 - crhs108*crhs132 - crhs110*crhs133 - crhs112*crhs134 - crhs113*crhs135 - crhs114*crhs132 - crhs115*crhs133 - crhs116*crhs134 - crhs117*crhs135 + crhs127 - crhs132*crhs77 + crhs132*crhs82 + crhs132*crhs88 - crhs133*crhs31 + crhs133*crhs90 + crhs133*crhs93 - crhs134*crhs74 + crhs134*crhs95 + crhs134*crhs98 - crhs135*crhs71 + crhs135*crhs99 - 0.5854102*crhs41 + crhs45 - 0.5854102*crhs51 + crhs55 - 0.5854102*crhs61 + crhs65 - 0.5854102*crhs71 + crhs75 + 0.19999999899376*f[2] + 0.40000000301872*f[3];
+
+
+    const double local_size = 4;
+    noalias(rRightHandSideVector) = rhs * rData.volume/local_size;
+
+    KRATOS_CATCH("");
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template <>
+void SymbolicQSConvectionDiffusionExplicit<2,3>::CalculateOrthogonalSubgridScaleRHSInternal(
+    BoundedVector<double, 3>& rRightHandSideVector,
+    const ProcessInfo& rCurrentProcessInfo)
+{
+    KRATOS_TRY;
+
+    // Element data
+    ElementData rData;
+    this->InitializeEulerianElement(rData,rCurrentProcessInfo);
+
+    // Compute tau
+    this->CalculateTau(rData);
+
+    // Retrieve element data
+    const auto& k = rData.diffusivity;
+    const auto& f = rData.forcing;
+    const auto& phi = rData.unknown;
+    const auto& phi_old = rData.unknown_old;
+    const auto& delta_time = rData.delta_time;
+    const auto& RK_time_coefficient = rData.RK_time_coefficient;
+    const auto& v = rData.convective_velocity;
```

Comment:
Just to be sure, are you considering the `MESH_VELOCITY` in the calculation of the convective velocity? If no, maybe it is a good idea to do it.

Reply:
Yes I do. See [here](https://github.com/KratosMultiphysics/Kratos/blob/symbolic-conv-diff-element-v2/applications/ConvectionDiffusionApplication/symbolic_generation/convection_diffusion_explicit/symbolic_qs_convection_diffusion_explicit_cpp_template.cpp#L256).